### PR TITLE
added profile API

### PR DESCRIPTION
**Pull request by avi4630**

### CODE REVIEW

File: node-backend/app/controllers/v1/profileController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const db = require("../../models/sequelize");
+const Users = db.users;
+
+const jwtToken = require("../../jwtTokenValidation/jwtValidation");
+module.exports.getProfile = async (req, res) => {
+  const authHeader = req.headers["authorization"];
+  let userData = await jwtToken.getData(authHeader);
```
Comment (sunil4sonawane) [inline node-backend/app/controllers/v1/profileController.js:7]: you can directly pass req.headers["authorization"] to function no need to get value in variable

File: node-backend/app/routes/profileRoutes.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+
+const /*eslint-disable no-unused-vars*/ profileControllerV1 = require("../controllers/v1/profileController");
```
Comment (sunil4sonawane) [inline node-backend/app/routes/profileRoutes.js:4]: @mayuriardad can you help on eslint at this line

File: node-backend/app/test/profile.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      .get("/profile")
+      .set("Authorization", "Bearer " + token)
+      .expect("Content-type", /json/)
+      .expect(200)
+      .end(function (err, res) {
+        res.status.should.equal(200);
+        done();
+      });
+  });
+  it("should give internal server error", function (done) {
```
Comment (sunil4sonawane) [inline node-backend/app/test/profile.test.js:31]: why internal server error?

File: node-backend/app/routes/profileRoutes.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+const utility = require("../utils/utility");
+const profileRouter = express.Router();
+profileRouter.use(bodyParser.urlencoded({ extended: true }));
+
+profileRouter.get("/profile", async (req, res) => {
+  let controller = await utility.getVersionedController(
+    req.headers,
+    "profileController"
+  );
+  /*eslint-disable no-eval*/ eval(controller).getProfile(req, res);
```
Comment (sunil4sonawane) [inline node-backend/app/routes/profileRoutes.js:14]: @sahilbhatia we are dynamically switching api version route with the help of eval function. Do you see any concerns int his?

Comment (sahilbhatia) [reply node-backend/app/routes/profileRoutes.js:14]: > Do you see any concerns int his?
Yes @sunil4sonawane. Using user input in eval is always a concern.

File: node-backend/app/controllers/v1/profileController.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+const db = require("../../models/sequelize");
+const Users = db.users;
+
+const jwtToken = require("../../jwtTokenValidation/jwtValidation");
+module.exports.getProfile = async (req, res) => {
+  let userData = await jwtToken.getData(req.headers["authorization"]);
+  Users.findByPk(userData.userId)
+    .then((profile) => {
+      res.status(200).send({
```
Comment (sunil4sonawane) [inline node-backend/app/controllers/v1/profileController.js:9]: check API doc for return values

**Pull request merged**